src/coffee/providers/map-loader.coffee

```diff
@@ -34,6 +38,25 @@ angular.module('uiGmapgoogle-maps.providers')
       isGoogleMapsLoaded = ->
         angular.isDefined(window.google) and angular.isDefined(window.google.maps)
 
+      onWindowLoad = (options)->
+        # if its a WebView
+        if !(!window.cordova && !window.PhoneGap && !window.phonegap && !window.forge)
```

Comment:
So overall this looks good. But the main problem I have with it is all this extra logic to keep supporting different environments all thrown into one factory. 

It would be nice if the onWindowLoad code actually comes from another provider, factory, or service where it can be overridden easily. I fear this will just keep growing.